app/controllers/admin/world_location_news_controller.rb

```diff
@@ -2,9 +2,13 @@ class Admin::WorldLocationNewsController < Admin::BaseController
   before_action :load_world_location, only: %i[edit update show features]
   layout :get_layout
 
-  def edit; end
+  def edit
+    render_design_system("edit", "legacy_edit", next_release: false)
+  end
 
-  def show; end
+  def show
```

Comment:
If this PR is about transitioning the edit page, why have we got code changes to the show action?